### PR TITLE
Add blog detail and tighten blog card component

### PR DESCRIPTION
**Pull request by mattleff**
Fixes https://trello.com/c/35A9SjjV/55-blog-component & https://trello.com/c/AIRH1EV4/62-blog-detail.

localhost_3000_english_blog_504 (3)

### Tablet

localhost_3000_english_blog_504 (2)

### CODE REVIEW

File: src/containers/blog/detail.graphql [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+		readingDuration
+		body
+	}
+	blogPosts(
+		language: $language
+		first: 5
+		orderBy: [{ field: PUBLISHED_AT, direction: DESC }]
+	) {
+		nodes {
+			...cardPost
```
Comment (narthur) [inline src/containers/blog/detail.graphql:19]: Would it be worth excluding the currently selected post in the card column at some point?

File: src/containers/blog/detail.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+} from '@pages/[language]/blog/[id]';
+
+jest.mock('@lib/writeFeedFile');
+
+const renderPage = buildStaticRenderer(BlogPostDetail, getStaticProps, {
+	language: 'en',
+	id: 'the_blog_post_id',
+});
+
+function loadData() {
```
Comment (narthur) [inline src/containers/blog/detail.spec.tsx:20]: In the future, `buildLoader` would probably work better for data loading: https://github.com/avorg/audioverse-next/blob/master/src/lib/test/helpers.tsx#L61

File: src/containers/blog/detail.tsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+				<div className={styles.sidebar}>
+					<LineHeading size={16}>
+						<FormattedMessage
+							id="blogDetailPage__moreBlogPosts"
+							defaultMessage="More Blog Posts"
+							description="Blog post detail more blog posts"
+						/>
+					</LineHeading>
+					<CardColumn
+						items={(blogPosts.nodes || [])?.map((post) => (
```
Comment (narthur) [inline src/containers/blog/detail.tsx:64]: Why the null coalescing if you're defaulting to an empty array?

**Review by narthur — APPROVED** [review]
Great work!

File: src/containers/blog/detail.graphql (hunk 1 above)
Comment (mattleff) [reply src/containers/blog/detail.graphql:20]: Good point: abd2530

File: src/containers/blog/detail.spec.tsx (hunk 2 above)
Comment (mattleff) [reply src/containers/blog/detail.spec.tsx]: Updated: abd2530

File: src/containers/blog/detail.tsx (hunk 3 above)
Comment (mattleff) [reply src/containers/blog/detail.tsx]: Yup 😉 abd2530

**Pull request merged**